src/algokit/core/log_handlers.py

```diff
@@ -27,7 +27,7 @@ class ClickHandler(logging.Handler):
     but they're kind of intertwined for our use case of actually displaying things to the user.
     """
 
-    styles: dict[str, dict[str, Any]] = {
+    styles: dict[str, dict[str, Any]] = {  # noqa: RUF012
```

Comment:
this one can also be `ClassVar` rather than suppressing.

Reply:
Resolved